qutip/nonmarkov/bofin_baths.py

```diff
@@ -752,3 +752,167 @@ def __init__(self, Q, ck_plus, vk_plus, ck_minus, vk_minus, tag=None):
                 "-", 2, Q, ckm, vkm, sigma_bar_k_offset=-1, tag=tag,
             ))
         super().__init__(exponents)
+
+
+class LorentzianBath(FermionicBath):
+    def __init__(self, Q, gamma, w, T, theta, lmax, tag=None):
+        mu_l = theta / 2.0
+        mu_r = - theta / 2.0
+
```

Comment:
One quick suggestion for both functions, as written here this parameterizes two baths (a L (left) and (R) right bath at chemical potentials mu_l and mu_r). Probably better if we just have these functions parameterize a single bath at chemical potential mu, and use the tags to differentiate them.

Reply:
Great idea -- implemented!